pkg/server/app/app.go

```diff
@@ -46,6 +47,7 @@ type K8sOperations interface {
 	SetNamespaceAnnotations(namespace string, annotations map[string]string) error
 	DeleteDeployEnvVars(namespace, name string, evNames []string) error
 	CreateOrUpdateDeployEnvVars(namespace, name string, evs []*EnvVar) error
+	FindAppByLabel(label string) ([]string, error)
```

Comment:
For the `k8s` package we were using names related of what we're looking for on the k8s api, f.ex `SetNamespaceAnnotations`.